src/style.css

```diff
@@ -1 +1,96 @@
-/* Styles go here */
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+}
+
+li {
+  list-style: none;
+}
+
+* {
+  all: unset;
+  display: revert;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: grey;
+  margin: 5px;
+}
+
+.conteiner {
+  display: flex;
+  background-color: white;
+  padding: 0 49px;
+  padding-right: 50px;
+  flex-direction: row;
+  justify-content: space-between;
+  flex-wrap: wrap;
+}
+
+.navigation {
+  display: flex;
+  position: relative;
+  flex-wrap: wrap;
+  height: auto;
+  padding-top: 19px;
+}
+
+.nav__img {
+  margin-top: 10px;
+  margin-bottom: 12px;
+}
+
+.nav__link {
+  position: relative;
+  font-size: 12px;
+  font-style: normal;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: black;
+  margin-left: 20px;
```

Comment:
You should avoid margin collapsing, when a child margins go through a parent and push off another parent
[useful_link](https://www.smashingmagazine.com/2019/07/margins-in-css/#parent-and-first-or-last-child-element)
So, if you want one element to be push off from another element use margins at THESE elements, not their children

Reply:
Comment above about making margin-left for nav__item insted here.
and set for nav-link display block or flex